internal/serve/httphandler/forgot_password_handler_test.go

```diff
@@ -370,6 +376,23 @@ func Test_ForgotPasswordHandler(t *testing.T) {
 		assert.JSONEq(t, wantsBody, string(respBody))
 	})
 
+	t.Run("Should return http status 401 when error getting tenant from context", func(t *testing.T) {
+		ctxWithoutTenant := context.Background()
+		requestBody := `
+        { 
+            "email": "valid@email.com" ,
+            "recaptcha_token": "validToken"
+        }`
```

Comment:
This indentation seems out of format.
```suggestion
		requestBody := `
                        { 
                                "email": "valid@email.com" ,
                                "recaptcha_token": "validToken"
                        }`
```